BankSDK/GiniBankSDKExample/GiniBankSDKExample/CustomOnboardingIllustrationAdapter.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  CustomOnboardingIllustrationAdapter.swift
+//  GiniBankSDKExample
+//
+//  Created by Nadya Karaban on 16.09.22.
+//
+
+import Foundation
+import GiniCaptureSDK
+import Lottie
+import UIKit
+class CustomOnboardingIllustrationAdapter: OnboardingIllustrationAdapter {
+    private var animatedOnboarding: AnimationView?
```

Comment:
[Suggestion] I get this warning in xCode:  AnimationView' is deprecated: `AnimationView` has been renamed to `LottieAnimationView`, for consistency with the new `LottieAnimation` type. This notice will be removed in Lottie 4.0.

Reply:
As I saw, Lottie 3.5 was released 7 days ago, that's when this warning was introduced.